webpack/scenes/ContentViews/Create/CreateContentViewForm.js

```diff
@@ -127,37 +139,51 @@ const CreateContentViewForm = ({ setModalOpen }) => {
       </FormGroup>
       <FormGroup isInline fieldId="type" label={__('Type')}>
         <Grid hasGutter>
-          <GridItem span={6}>
+          <GridItem span={4}>
             <Tile
               style={{ height: '100%' }}
               isStacked
               aria-label="component_tile"
-              icon={<ContentViewIcon composite={false} />}
+              icon={<ContentViewIcon composite={false} rolling={false} />}
               id="component"
               title={__('Content view')}
-              onClick={() => { setComponent(true); setComposite(false); }}
+              onClick={() => { setComponent(true); setComposite(false); setRolling(false); }}
               isSelected={component}
             >
-              {__('Single content view consisting of e.g. repositories')}
+              {__(contentViewDescriptions.CV)}
```

Comment:
Not sure this translation will work..Better to translate strings in contentViewDescriptions directly.

Reply:
It won't work.

Reply:
Marking a review on this so we don't forget.

Reply:
I believe this is still resolved with the latest push.